web/src/lib/domain-logic/starterKits.test.ts

```diff
@@ -0,0 +1,64 @@
+import { EssentialQuestions } from "@/lib/domain-logic/essentialQuestions";
+import { createStarterKitProfileData } from "@/lib/domain-logic/starterKits";
+import { emptyProfileData, Industries, ProfileData } from "@businessnjgovnavigator/shared";
+
+describe("starter Kits", () => {
+  const combineObjectsKeepingTruthyValues = (obj1: ProfileData, obj2: ProfileData): ProfileData => {
+    let result: ProfileData = { ...obj1 };
+    for (const key in obj2) {
+      const typedKey = key as keyof ProfileData;
+      if (!result[typedKey]) {
+        result = {
+          ...result,
+          [typedKey]: obj2[typedKey],
+        };
+      }
+    }
+    return result;
+  };
+
+  const removeFalsyValues = (obj: ProfileData): Partial<ProfileData> => {
+    let result: Partial<ProfileData> = {};
+    for (const key in obj) {
+      const typedKey = key as keyof ProfileData;
+      if (Object.prototype.hasOwnProperty.call(obj, typedKey) && obj[typedKey]) {
+        result = {
+          ...result,
```

Comment:
dust - you can add logic to in the conditional to see if the value is false and use the delete keyword to remove the key

Reply:
I'm pretty sure I can't because the field is read only